src/altinn-app-frontend/src/features/form/components/MessageBanner.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { AltinnAppTheme } from 'altinn-shared/theme';
+
+import MessageBanner from './MessageBanner';
+
+describe('form/components/MessageBanner.tsx', () => {
+  let mockLanguage: any;
+  let mockMessageKey: string;
+
+  beforeEach(() => {
+    mockMessageKey = 'form_filler.required_description';
+    mockLanguage = {
+      'form_filler': {
+        'required_description': 'Obligatoriske felter er merket med *',
+      }
+    }
+  });
```

Comment:
```suggestion
  const mockMessageKey = 'form_filler.required_description';
  const mockLanguage = {
    'form_filler': {
      'required_description': 'Obligatoriske felter er merket med *',
    }
  }
```

Maybe we need to add the `ILanguage `interface for the mock lang to work withouth the 'any' cast.